conf/ui.yaml.template

```diff
@@ -0,0 +1,38 @@
+UI:
+  SCREENSHOTS_PATH: /tmp/robottelo/screenshots/
+  # Webdriver logging options
+  LOG_DRIVER_COMMANDS:
+  - newSession
+  - windowMaximize
+  - get
+  - findElement
+  - sendKeysToElement
+  - clickElement
+  - mouseMoveTo
+
+  # browser tells robottelo which browser to use when testing UI. Valid values (selenium/remote)
+  BROWSER: selenium
+  # Webdriver to use
+  WEBDRIVER: chrome
+  WEBDRIVER_BINARY: /usr/bin/chromedriver
+  # Desired Capabilities of Web Driver
+  WEBDRIVER_DESIRED_CAPABILITIES:
```

Comment:
These all repeat what is in the `webkaifuku` block.

Reply:
I was thinking to move that webkaifuku